build/lib/scripts/enable-kiosk

```diff
@@ -58,6 +58,7 @@ while ! curl "http://localhost" > /dev/null; do
     sleep 1
 done
 matchbox-window-manager -use_titlebar no &
+xdotool search --sync --onlyvisible --class "Firefox" windowactivate key F11 &
```

Comment:
I would use a variable for the class name.

Reply:
ok look again 👇